### PR TITLE
Update Health Intro

### PR DESCRIPTION
**Pull request by spetrosky-va**
Updated Health Introduction to include paragraph about production health data hold. https://vajira.max.gov/browse/API-1780

### Process

- [ ] Tests added or updated for change
- [ ] Docs updated
- [ ] Accessibility concerns have been considered and addressed

### CODE REVIEW

**Comment by spetrosky-va** [comment]
Added release notes to this pull request.

File: src/apiDefs/data/health.ts [hunk 1]
```diff
@@ -90,7 +90,8 @@ const healthApis : IApiDescription[] = [
     oAuth: true,
     releaseNotes: FhirApiReleaseNotes,
     tabBlurb: 
-      "The VA's FHIR Health APIs allow consumers to develop applications using Veteran data. Please see the tabs below for the specific FHIR implementations.",
+      "The VA's FHIR Health APIs allow consumers to develop applications using Veteran data. Please see the tabs below for the specific FHIR implementations. 
+      Data entered through the production environment Veterans Health API is held in the original data source for 36 hours before it is visible elsewhere, including in any patient-facing applications. This holding period exists to allow health care providers time to discuss health data, such as sensitive diagnoses, before the patient sees this data elsewhere.",
```
[the remark this answers is not in the document]
Comment (spetrosky-va) [reply src/apiDefs/data/health.ts:94]: @mjlumetta yes, please.

[the remark this answers is not in the document]
Comment (mjlumetta) [reply src/apiDefs/data/health.ts]: @spetrosky-va done. it's ready for you to review the content. if you're happy with it, it's ready to merge.

File: src/containers/documentation/ApiDocumentation.tsx [hunk 2]
```diff
@@ -55,32 +60,30 @@ class ApiDocumentation extends React.Component<IApiDocumentationProps, IApiDocum
...
-                  })}
-                </TabList>
+        {apiDefinition.docSources.length === 1 ? (
+          <SwaggerDocs
+            docSource={apiDefinition.docSources[0]}
+            apiName={apiDefinition.urlFragment}
+          />
+        ) : (
+          <React.Fragment>
+            {apiDefinition.multiOpenAPIIntro && apiDefinition.multiOpenAPIIntro({})}
```
Comment (rtravitz) [inline src/containers/documentation/ApiDocumentation.tsx:70]: What's the purpose of the invocation with the empty object after checking the the `multiOpenAPIIntro` exists?

Comment (mjlumetta) [reply src/containers/documentation/ApiDocumentation.tsx:70]: it requires the props object as a React component. you'll get a TS error otherwise.

**Comment by mjlumetta** [comment]
@rtravitz yes, that makes sense. will circle back when i have a chance.

**Pull request merged**